fs-storage/src/base_storage.rs

```diff
@@ -1,7 +1,9 @@
 use data_error::Result;
+use jnix_macros::{FromJava, IntoJava};
 use std::collections::BTreeMap;
 
-#[derive(Debug, PartialEq, PartialOrd, Ord, Eq, Clone)]
+#[derive(Debug, PartialEq, PartialOrd, Ord, Eq, Clone, FromJava, IntoJava)]
+#[jnix(class_name = "SyncStatus")]
```

Comment:
Use a conditional derive for `FromJava` and `IntoJava`. This is so that Java bindings are completely featured flagged and it's possible to turn it off if a user only wants to import the Rust core.

```
#[cfg_attr(feature = "<feature>", derive(Stuff))]
```

Similarly for the `jnix_macros` import.

Reply:
done